zipkin-server/src/main/java/zipkin/server/brave/BraveConfiguration.java

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright 2015-2017 The OpenZipkin Authors
+ * Copyright 2015-2018 The OpenZipkin Authors
```

Comment:
ignore this.. just didn't want to litter with a commit update to satisfy license plugin